.travis.yml

```diff
@@ -6,8 +6,8 @@ before_install:
 - sudo apt-key adv --keyserver packages.microsoft.com --recv-keys 417A0893
 - sudo apt-get install -y apt-transport-https
 - sudo apt-get update && sudo apt-get install -y azure-cli
+- sudo apt-get install -y jq
 install:
-- npm install
```

Comment:
Why was this removed?

Reply:
Ah, of course, because we don't have any npm dependencies anymore :)

Reply:
Correct :)